### ISSUE TITLE
Missing filename in error output

### ISSUE THREAD

**Issue by ashemedai** [awslabs/cfn-python-lint#142]
Using cfn-lint 0.3.0 I was trying to fix up linting on vim.

At some point while testing the linting I deleted a three W3010 warnings from this file (explicit mentions of Properties/AvailabilityZones under an RDSCluster node) and this caused cfn-lint to output the follow from the lint invocation: `:159:25:159:26:E0000:Null value at line 158 column 24`. As you can see, the filename is missing.

So it boils down to:

```yaml
RDSCluster:
  Type: "AWS::RDS::DBCluster"
  Properties:
    AvailabilityZones:
    BackupRetentionPeriod: 35
```

So in essence if a sequence of scalars is empty it will return this linting line, which misses the filename.

Also tested this with a `Resources/AutoScalingGroup/Properties/VPCZoneIdentifier` by removing all its scalars.

**Comment by ashemedai** [comment]
Looks like something triggers `ParserError` and is not setting the filename properly.

https://github.com/awslabs/cfn-python-lint/blob/262861c170854f91ab91c9098f606b203a352fa5/src/cfnlint/__init__.py#L813-L818.

**Comment by ashemedai** [comment]
Tracked it down to:

https://github.com/awslabs/cfn-python-lint/blob/41df652edcbaf4fcc62958b7ddf9f12ef1cc25f3/src/cfnlint/cfn_yaml.py#L51-L53

What confuses me is the fact line numbers and column numbers are present though.

**Comment by kddejong** [comment]
Fixed and released with v0.3.1.
